src/libraries/System.Net.Http/tests/StressTests/HttpStress/Dockerfile

```diff
@@ -7,6 +7,18 @@ RUN echo "DOTNET_VERSION="$DOTNET_VERSION
 WORKDIR /app
 COPY . .
 
+# Pulling the msquic Debian package from msquic-ci public pipeline and from a hardcoded build.
+# Note that this is a temporary solution until we have properly published Linux packages.
+# Also note that in order to update to a newer msquic build, you have update this link.
```

Comment:
Does this link already contain changes needed for https://github.com/dotnet/runtime/pull/55291? If not, can we update it?

Reply:
Yes it does contain them.